src/ID3Helpers.js

```diff
@@ -128,20 +128,60 @@ function getFramesFromID3Body(ID3FrameBody, ID3Version, options = {}) {
             continue
         }
         const frameHeaderFlags = ID3Util.parseFrameHeaderFlags(bodyFrameHeader, ID3Version)
+        if(frameHeaderFlags.dataLengthIndicator) {
+            bodyFrameSize -= 4
```

Comment:
I don't understand why this is `-4` isn't the header longer by 4 bytes when compressed?

Reply:
Yes, the header is longer by 4 bytes, but we have to take those 4 bytes away from the body, because normally, we assume that the body is of size (frame.length - 10) instead of (frame.length - 14).

The code is a bit confusing, I'll try to make it more understandable

Reply:
I've changed the code to make it a bit more clear and added comments but I chose not to add the data length indicator into the header because technically it is not part of the header

> This information is added after the frame header and before the frame data